Swift/BRCryptoDemo/TransferViewController.swift

```diff
@@ -251,9 +251,6 @@ class TransferViewController: UIViewController, TransferListener {
                     break // nearly impossible
                 }
                 break
-            case .confirmation(let count):
-                self.confCountLabel.text = count.description
-                break
             case .deleted:
```

Comment:
So arguably `TransferViewController` needs to be a `WalletManagerListener` too (and register as such) and handle a BLOCK_HEIGHT_UPDATED event by updating the `confCountLabel`?

Reply:
Updated to account for this.